src/Config/ObjectModel/Entity.cs

```diff
@@ -20,4 +20,8 @@ public record Entity(
     EntityRestOptions Rest,
     EntityPermission[] Permissions,
     Dictionary<string, string>? Mappings,
-    Dictionary<string, EntityRelationship>? Relationships);
+    Dictionary<string, EntityRelationship>? Relationships)
+{
+    public const string PROPERTY_PATH = "path";
```

Comment:
these are properties of `EntityRestOptions`, can't we do `nameof(EntityRestOptions.Path)` `nameof(EntityRestOptions.Methods)`?

Reply:
The properties in `EntityRestOptions` start with uppercase P/M. We need lowercased characters in the property names. Using `nameof(EntityRestOptions.Path).ToLower()` makes the expression non-constant. So, we cannot do it. Also, we cannot rename the property from Path->path as the property name has to start with a capital letter - that's a naming rule.

Reply:
Where is `PROPERTY_PATH` string used? I couldn't find it when I searched. Is it for error messaging only? Do we really need a `const` string? 

Also, if at all we still need it, wouldn't `EntityRestOptions` a better class to have these defined?

Reply:
Moved the properties to the entityrestoptions class